tasking/pulp/tasking/celery_app.py

```diff
@@ -92,6 +94,7 @@ def get_resource_manager_lock(name):
     :param name:   The hostname of the worker
     :type  name:   basestring
     """
+    from pulp.app.models.task import TaskLock, Worker
```

Comment:
Can we do this runtime lookup using the Django machinery. Can we try this instead?

```
from django.db.models import get_app
pulp_app =  get_app('pulp_app')
```

Then you could use `pulp_app.models.TaskLock` and `pulp_app.models.Worker`

I haven't tested this but I think it would work.